src/components/forms/Label.js

```diff
@@ -1,19 +1,21 @@
 import {FormLabel, Paragraph} from '@utrecht/component-library-react';
+import classNames from 'classnames';
 import PropTypes from 'prop-types';
 import React from 'react';
 
-import {getBEMClassName} from 'utils';
-
-const Label = ({id, isRequired = false, disabled = false, children}) => {
-  const modifiers = isRequired ? ['required'] : [];
-  return (
-    <Paragraph className={getBEMClassName('label', modifiers)}>
-      <FormLabel htmlFor={id} disabled={disabled}>
-        {children}
-      </FormLabel>
-    </Paragraph>
-  );
-};
+const Label = ({id, isRequired = false, disabled = false, children}) => (
+  <Paragraph className="utrecht-form-field__label">
+    <FormLabel
+      htmlFor={id}
+      disabled={disabled}
+      className={classNames({
+        'utrecht-form-label--openforms-required': isRequired,
```

Comment:
Does this take into account the special DH style for required fields (plain for required fields and followed by 'optioneel' for optional fields)?

Reply:
I've lifted up this configuration into the `ConfigContext` which is initialized/set in the `sdk.js`. This exposes it to the `Label` and new `Wrapper` components. I've also updated the CSS to apply this style to the utrecht components instead of only our own.